src/test/java/org/opensearch/neuralsearch/processor/TextChunkingProcessorIT.java

```diff
@@ -176,6 +181,33 @@ public void testTextChunkingProcessor_withCascadePipeline_successful() {
         }
     }
 
+    @SneakyThrows
+    public void testTextChunkingProcessor_withIgnoreMissing() {
+        try {
+            createPipelineProcessor(IGNORE_MISSING_PIPELINE_NAME);
+            createTextChunkingIndex(INDEX_NAME, IGNORE_MISSING_PIPELINE_NAME);
+            ingestDocument(TEST_MISSING_FIELD_DOCUMENT);
+
+            validateIndexIngestResults(INDEX_NAME, OUTPUT_FIELD, null);
```

Comment:
This IT fails in this line:

`assert (documentSourceMap).containsKey(fieldName);`

If the expected is null, please assert documentSourceMap do not contain fieldName

Reply:
That makes sense. Seems that my local tests were not failing because Gradle was using an old cached build? idk. But it is now fixed

Reply:
IDK how you run the integration test. I would suggest that you could run integration test in intellij IDEA.